circle.yml

```diff
@@ -1,13 +1,19 @@
 dependencies:
+  pre:
+    - curl -o- -L https://yarnpkg.com/install.sh | bash
```

Comment:
I ran `curl --location https://yarnpkg.com/install.sh | cat` and got the expected result. :)